internal/helmutil/index_v2.go

```diff
@@ -121,6 +121,10 @@ func (idx *IndexV2) SortEntries() {
 	idx.index.SortEntries()
 }
 
+func (idx *IndexV2) TimeStamp() {
```

Comment:
Perhaps we should clarify the name?

```suggestion
func (idx *IndexV2) UpdateGeneratedTime() {
```

Reply:
Agree